### ISSUE TITLE
Ender I/O Travel Anchor Bug

### ISSUE THREAD

**Issue by MonfGeiger** [Sunekaer/stoneBlock#698]
## Bug Report
Every-time I re-log into complex gaming stoneblock, the 2nd ender i/o travel anchor I placed vanishes from the travel points, forcing me to dig down to the 2nd, break it, place it, or walk over it for a bit, but the first one I placed never bugs

## Steps to Reproduce (for bugs)
1. Put down one travel anchor on the floor
2. dig pretty far down and make a small cavern, then place a second travel anchor
3. travel back to the 1st, and log off the server and back on

## Logs
I have tried multiple times to upload the logs to paste.feed-the-beast, but it never shows it right, says it can't connect to the website, or uploads and only shows a blank white screen. I have tried 5 times and this keeps happening, so if you would PLEASE accept an alternate way of showing the logs, that would be nice. Maybe regular pastebin?

paste.feed-the-beast.com is completely unresponsive on any other browser I try

## Client Information
* Modpack Version: 1.0.31
* Server/LAN/Single Player: Server
* Optifine Installed: no
* Shaders Enabled: no

**Comment by Sunekaer** [comment]
Does it only happen on this server?

**Comment by MonfGeiger** [comment]
so far; I don't know if this is intended or not, but I had to dig a hole down to the second anchor, and then when I relogged, it all worked

for some reason, in the stoneblock mc complex gaming server, my second anchor needs a hole connecting to the same room as the 1st one; if the hole is filled in, upon relog, it breaks again. If there's no way to fix it beyond bringing this issue to the ender i/o dev, ill switch to viaducts instead

**Comment by Sunekaer** [comment]
yeah not anything i can do about this, it would be the EnderIO team there should look at it
